pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/AuthenticationRoute.java

```diff
@@ -89,24 +92,25 @@ public Object handle(Request request, Response response) {
                         logger.info("User (" + email + ") was found ");
                         Gson gson = new Gson();
```

Comment:
there is a singleton class called `ObjectMapperSingleton` which is wrapper of `jackson`, let's use it everywhere instead of `gson`